src/dex-ui/pages/dao/utils.ts

```diff
@@ -98,3 +106,31 @@ export function getProposalSteps(
   }
   return { steps, activeStep };
 }
+
+export function getProposalData(proposal: Proposal): string {
+  if (proposal.type === ProposalType.AddNewMember) {
+    const { owner, _threshold } = proposal.data as ProposalDataAddMember;
+    console.log("[ANS] owner", proposal.data, owner, proposal);
+    const ownerAddress = owner ? solidityAddressToTokenIdString(owner) : "";
+    return `Proposed to add new member ${ownerAddress} and change threshold to ${_threshold}`;
+  }
+  if (proposal.type === ProposalType.RemoveMember) {
+    const { owner, _threshold } = proposal.data as ProposalDataDeleteMember;
+    const ownerAddress = solidityAddressToTokenIdString(owner);
+    return `Proposed to remove member ${ownerAddress} and change threshold to ${_threshold}`;
+  }
+  if (proposal.type === ProposalType.ReplaceMember) {
+    const { oldOwner, newOwner } = proposal.data as ProposalDataReplaceMember;
+    const oldOwnerAddress = solidityAddressToTokenIdString(oldOwner);
```

Comment:
The same issue with the `solidityAddressToTokenIdString` function described above may occur here as well.